src/main/java/sirius/biz/storage/layer2/Blob.java

```diff
@@ -207,16 +203,18 @@
      *
      * @return a list of all known variants
      */
-    List<BlobVariant> getVariants();
+    List<? extends BlobVariant> fetchVariants();
 
     /**
-     * Lists all known revisions of this blob.
+     * Tries to find the variant with the given name.
      * <p>
-     * A revision is a backup copy of previously provided data of this blob.
+     * This will check if a variant with the given name exists and return it. If no such varian exists,
```

Comment:
... If no such variant **exists**, ...

_Sorry, suggestions cannot be applied_